code/modules/antagonists/blob/blob_mobs.dm

```diff
@@ -225,6 +226,12 @@
 		color = initial(color)//looks better.
 		add_overlay(blob_head_overlay)
 
+/mob/living/simple_animal/hostile/blob/blobspore/add_cell_sample()
+	AddElement(/datum/element/swabable, CELL_LINE_TABLE_BLOBSPORE, CELL_VIRUS_TABLE_GENERIC_MOB, 1, 5)
+
+/mob/living/simple_animal/hostile/blob/blobspore/independent
```

Comment:
is it intentional for this subtype to be in the gold slime spawning pool?

Reply:
Yeah, i think so.